client/src/components/Shows/ShowCalendar.js

```diff
@@ -109,24 +115,29 @@ class Calendar extends Component {
   };
 
   render() {
-    const { shows } = this.props;
+    const { date, shows } = this.props;
     const localizer = BigCalendar.momentLocalizer(moment);
 
+    //if date provided in properties, always have the calendar display that date
+    //otherwise, just start with today
+    let calendarDateProperty =
+      typeof date == 'undefined' ? { defaultDate: new Date() } : { date: date };
```

Comment:
I think defaultDate can always stay on the calendar and then this is overwritten by the date when you pass it.  I had problems with changing dates in the past without defaultDate so I'm not sure if leaving it off makes a difference once we apply the real date.